packages/roger/watch.ts

```diff
@@ -104,6 +109,7 @@ export default async function (port = 9160): Promise<void> {
                           : relative(".", resolve(parentDir, style[0])), // HACK: use the first style path as the path to the combined Style because the combined Style doesn't exist in the file system
                     },
                   },
+                  excludeWarnings,
```

Comment:
```suggestion
                  excludeWarnings ?? [],
```

Reply:
This works: `excludeWarnings: excludeWarnings ?? []`.
Done